spring-cloud-kubernetes-examples/kubernetes-leader-election-example/README.md

```diff
@@ -0,0 +1,83 @@
+# Spring Cloud Kubernetes leader election example
+
+## Setting up the Environment
+
+This example uses a Fabric8 Maven Plugin to deploy an application to a Kubernetes cluster.
+To try it locally, download and install `[Minikube](https://kubernetes.io/docs/getting-started-guides/minikube/)`.
+
+Once Minikube is downloaded, start it with the following command:
+```
+minikube start
+```
+
+And configure environment variables to point to the Minikube's Docker daemon:
+```
+eval $(minikube docker-env)
+```
+
+## Overview
+
+Spring Cloud Kubernetes leader election mechanism implements leader election API of Spring Integration using Kubernetes ConfigMap.
```

Comment:
Can this technology only be used with `Spring Integration applications` ? What about Apache Camel ?

Reply:
Can you then add a link within the example to point to the Spring Integration code, doc, .... introducing the leader mechanism ?